loky/backend/resource_tracker.py

```diff
@@ -74,15 +71,11 @@
 VERBOSE = False
 
 
-class ResourceTracker:
-    def __init__(self):
-        self._lock = threading.Lock()
-        self._fd = None
-        self._pid = None
-
-    def getfd(self):
+class ResourceTracker(_ResourceTracker):
+    def maybe_unlink(self, name, rtype):
+        """Decrement the refcount of a resource, and delete it if it hits 0"""
         self.ensure_running()
-        return self._fd
+        self._send("MAYBE_UNLINK", name, rtype)
 
     def ensure_running(self):
```

Comment:
Too bad that we cannot tweak the cmd from the base class.
We should maybe say why we keep this?

Reply:
I will add a docstring to our `ResourceTracker` class.

Reply:
Done in 4ead44609f1467c4ff0a6502017186c7e70bbe45.